nailgun/entities.py

```diff
@@ -2361,6 +2363,25 @@ def create_payload(self):
             u'operatingsystem': super(OperatingSystem, self).create_payload()
         }
 
+    def update(self, fields=None):
```

Comment:
Just to be clear, I dropped the `update` method when creating the sample code above.